axelar-chains-config/info/testnet.json

```diff
@@ -2071,6 +2071,77 @@
     "rpc": "https://rpc-axelar-testnet.imperator.co:443",
     "lcd": "https://lcd-axelar-testnet.imperator.co:443",
     "grpc": "grpc-axelar-testnet.imperator.co:2062",
+    "tokenSymbol": "AXL"
+  },
+  "sui": {
+    "axelar": {
```

Comment:
scope all packages under `"contracts"` key like evm contracts, in this case just empty since there's no official deployment